docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
       REDASH_LOG_LEVEL: "INFO"
       REDASH_REDIS_URL: "redis://redis:6379/0"
       REDASH_DATABASE_URL: "postgresql://postgres@postgres/postgres"
+      REDASH_THROTTLE_LOGIN_ENABLED: "false"
```

Comment:
Not `"true"`?

Reply:
This one is for dev :). Almost sure you don't want a login limit here

Reply:
So this would be the prod setting?
https://github.com/getredash/redash/blob/5760ef13eb2c9d5bbaf9eb26f7f34892aafce722/redash/settings/__init__.py#L153

Reply:
> So this would be the prod setting?

That's the default setting, thus prod setting as prod doesn't have this set

Reply:
Got it 👍
The syntax here is confusing. I thought it works like `lodash.get`.